packages/mm/mm.0.7.4/opam

```diff
@@ -0,0 +1,44 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "The mm library contains high-level to create and manipulate multimedia streams (audio, video, MIDI)"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Romain Beauxis <toots@rastageeks.org>"]
+license: "GPL-2.0"
+homepage: "https://github.com/savonet/ocaml-mm"
+bug-reports: "https://github.com/savonet/ocaml-mm/issues"
+depends: [
```

Comment:
```suggestion
depends: [
  "ocaml" {with-test & >= "4.07"}
```
```
#=== ERROR while compiling mm.0.7.4 ===========================================#
# context              2.0.10 | linux/x86_64 | ocaml-base-compiler.4.03.0 | pinned(https://github.com/savonet/ocaml-mm/archive/v0.7.4.tar.gz)
# path                 ~/.opam/4.03/.opam-switch/build/mm.0.7.4
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p mm -j 31 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/mm-64-5f1023.env
# output-file          ~/.opam/log/mm-64-5f1023.out
### output ###
#          gcc src/image_rgb.o
# image_rgb.c: In function 'caml_rgb_color_to_alpha':
# image_rgb.c:1044:10: warning: variable 'd' set but not used [-Wunused-but-set-variable]
#  1044 |   double d;
#       |          ^
# image_rgb.c:1043:22: warning: variable 'aa' set but not used [-Wunused-but-set-variable]
#  1043 |   double rr, gg, bb, aa;
#       |                      ^~
#       ocamlc examples/.test.eobjs/byte/dune__exe__Test.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.03/bin/ocamlc.opt -w -40 -g -bin-annot -I examples/.test.eobjs/byte -I src/.mm_audio.objs/byte -I src/.mm_base.objs/byte -I src/.mm_image.objs/byte -no-alias-deps -o examples/.test.eobjs/byte/dune__exe__Test.cmo -c -impl examples/test.ml)
# File "examples/test.ml", line 62, characters 41-53:
# Error: Unbound module Float
```

Reply:
Thanks! Commited and applied upstream as well.